config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml

```diff
@@ -206,7 +206,7 @@ presubmits:
         - --ginkgo-parallel=40
         - --provider=gce
         - --stage=gs://kubernetes-release-pull/ci/pull-kubernetes-e2e-gce-correctness
-        - --test_args=--ginkgo.skip=\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\] --minStartupPods=8 --node-schedulable-timeout=90m
+        - --test_args=--ginkgo.skip=\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:([^L].*|L[^o].*|Lo[^a].*|Loa[^d].*)\] --minStartupPods=8 --node-schedulable-timeout=90m
```

Comment:
This regexp is... weird :-) I wish we have something better/more readable.

But it makes sense to merge it for consistency for now.

Reply:
we have been discussing this for a week , the problem is that golang regexp does not have lookahead, but that does not remove the complexity, I created this doc to brainstorm an idea of using key.value pair for tags on tests , please chime in https://docs.google.com/document/d/1zdj-QGfxwlY39Q9hPHDkZGM4DsJbWobNZz87lxBmboE/edit